src/consts.js

```diff
@@ -282,6 +253,30 @@ export const ENV_VARS = {
     CONTAINER_URL: 'APIFY_CONTAINER_URL',
 };
 
+/**
+ * Local emulation sub directories for local stores
+ */
+export const LOCAL_STORAGE_SUBDIRS = {
+    datasets: 'datasets',
+    keyValueStores: 'key-value-stores',
+    requestQueues: 'request-queues',
```

Comment:
This is same as current, but shouldn't we use either all underscores or all dashes?

`apify_storage` vs `key-value-stores`

Makes it difficult to write paths without checking syntax first.

It looks good otherwise! Are you gonna do the SDK PR that sets the variables too?